src/pkg/blob/dao/dao.go

```diff
@@ -218,38 +217,12 @@ func (d *dao) UpdateBlob(ctx context.Context, blob *models.Blob) error {
 	return err
 }
 
-func (d *dao) ListBlobs(ctx context.Context, params models.ListParams) ([]*models.Blob, error) {
-	qs, err := orm.QuerySetter(ctx, &models.Blob{}, nil)
+func (d *dao) ListBlobs(ctx context.Context, query *q.Query) ([]*models.Blob, error) {
+	qs, err := orm.QuerySetter(ctx, &models.Blob{}, query)
```

Comment:
Process the sorting in the query when want to fix the loop problem in the gc task.